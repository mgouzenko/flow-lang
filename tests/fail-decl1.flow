// Should fail as bad type

int main() {
  doudffhd x; // Fail
}