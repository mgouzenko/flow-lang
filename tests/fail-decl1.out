NEED SOME SORT OF FAILURE